beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -38,15 +38,12 @@ const counterSeconds = 20
 func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-
 	counter := ratecounter.NewRateCounter(counterSeconds * time.Second)
-
 	var lastEmptyRequests int
 	errChan := make(chan error)
 	// Step 1 - Sync to end of finalized epoch.
-	for s.chain.HeadSlot() < helpers.StartSlot(highestFinalizedEpoch()+1) {
```

Comment:
Please revert this. We want to sync to the start of the next epoch after finalized.

Reply:
+1

Reply:
```DEBU[0003] helpers.StartSlot(highestFinalizedEpoch()+1)-1: 127  head status:=124 prefix=initial-sync```

Reply:
```DEBU[0003] Requesting blocks...                          peer=12D3KooWE4qDcRrueTuRYWUdQZgcy7APZqBngVeXRt4Y6ytHizKV prefix=initial-sync req="head_block_root:\"finalized_root 1\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\\000\" start_slot:114557 count:64 step:4 "
```

Reply:
it seems strange as this is the code that calls it ```blocks, err := request(
			s.chain.HeadSlot()+1, // start
			1,                    // step
			blockBatchSize,       // count
			peers,                // peers
			0,                    // remainder
		)```

Reply:
found the issue + fixed